docs/interactions/Message_Components.md

```diff
@@ -14,28 +14,33 @@ The top-level `components` field is an array of [Action Row](#DOCS_INTERACTIONS_
+| max_values?  | integer                                                                                                     | the maximum number of items that can be chosen; default 1, max 25                         | [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus)                                                                                                                                                                                                    |
+| components?  | array of [components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object)                               | a list of child components                                                                | [Action Rows](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/action-rows)                                                                                                                                                                                                      |
+| title?       | string                                                                                                      | title of the modal                                                                        | [Interaction Callback Type Modal](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object-interaction-callback-type)                                                                                                                                  |
+| min_length?  | integer                                                                                                     | the minimum length of the text input                                                      | [Input Text](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/input-text)                                                                                                                                                                                                        |
+| max_length?  | integer                                                                                                     | the maximum length of the text input                                                      | [Input Text](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/input-text)                                                                                                                                                                                                        |
+| placeholder? | string                                                                                                      | the placeholder for the text input field                                                  | [Input Text](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/input-text)                                                                                                                                                                                                        |
```

Comment:
can't it be in the `placeholder?` above? or it doesn't have the limit of 100 characters?

Reply:
Ah you're right. Fixing this!